build.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
```

Comment:
This file should probably go into the `docker` folder

Reply:
I used `cd $(dirname "${BASH_SOURCE[0]}")`, so the script kind-of revolves around being in the root dir. Should I append `..` or should I use `git rev-parse --show-toplevel` to get the root directory instead?

Reply:
`git rev-parse --show-toplevel` seems like the way to go to me, but I'd rather defer to @madebr or @Ramen2X who might be more familiar with builds or the Docker situation.

I believe until now we have generally avoided including build scripts/helpers in the repo, so I can't say whether we'll want to move forward with this in general or not.

Reply:
Yeah, I get that. This isn't needed at all, just nice-to-have.